exatomic/core/atom.py

```diff
@@ -105,6 +139,8 @@ def rotate(self, theta, axis=None, frame=None, degrees=True):
         if axis is None: axis = [0, 0, 1]
         if frame is None: frame = self.last_frame.copy()
         else: frame = self[self.frame == frame].copy()
+
+        if all(list(map(lambda x: x == 0., axis))) or theta == 0.: return frame
```

Comment:
Does `all(map(lambda x: x...))`?

Reply:
Yeah, that would probably look better. I pushed the updates. Thank you!